woocommerce/woocommerce
# CYS: Fix color button

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR fixes the button color for dark patterns in the Neutral palette.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

0. Go to `wp-admin/tools.php?page=woocommerce-admin-test-helper`, click the `Features` tab, and enable the `pattern-toolkit-full-composability` feature flag.
1. Go to `wp-admin/admin.php?page=wc-settings&tab=advanced&section=woocommerce_com` and uncheck `Allow usage of WooCommerce to be tracked`
2. Now recheck it and save.
3. Go to `wp-admin/tools.php?page=action-scheduler`, search for the `fetch_patterns` action, and ensure it was scheduled.
4. Click on `Run` under the action name if it is still pending and wait until it is completed.
5. Go to `wp-admin/admin.php?page=wc-admin&path=%2Fcustomize-store`, click on `Start designing` or `Customize your theme`.
6. In the assembler, click on `Design your homepage`.
7. Click on the `Intro` category.
8. Check all the buttons on dark patterns are white.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
CYS - Fix dark patterns buttons color.
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
